src/App.js

```diff
@@ -1,9 +1,15 @@
+import * as S from './styles'
 
 function App() {
+  const arr = ['(', ')', '%', 'AC', 7, 8, 9, '%', 4, 5, 6,'X', 1, 2, 3, '-', 0, '.', '=', '+'];
+  
   return (
-    <div className="App">
-      
-    </div>
+    <S.Container>
+      <S.InputWrap></S.InputWrap>
+      <S.BtnContainer>
+        {arr.map((str, i)=>(<S.Btn isNum={typeof(str)==='number'} key={i}>{str}</S.Btn>))}
```

Comment:
str 대신 좀더 의미있는 이름을 쓰시면 좋을같습니다!

Reply:
```suggestion
        {arr.map((str, i)=>(
          <S.Btn key={i} isNum={typeof(str)==='number'}>{str}</S.Btn>
        ))}
```

라인이 너무 길면 가독성이 떨어지기때문에, 한줄 띄어주시면 가독성에 더 좋을거 같습니다 :) 
+ 저는 key 값은 맨 앞에 적는게 좋더라구요!